### PR TITLE
possible fix: fatal error LNK1169, one or more multiply defined symbo…

### PR DESCRIPTION
**Pull request by smalti**
Fix applied for `MOCK_MODULE_FUNC0_`

### CODE REVIEW

**Review by Alex0vSky — DISMISSED** [review]
A good solution to the problem. Ready to remove #17

**Comment by smalti** [comment]
> A good solution to the problem. Ready to remove https://github.com/smalti/gmock-win32/pull/17

Ok, so I'll apply the fix for other `MOCK_MODULE_FUNC**_` macro-defines

File: include/gmock-win32.h [hunk 1]
```diff
@@ -50,6 +50,22 @@ namespace detail {
...
+            return obj;
+        }
+
+        static void** ppOldFn()
+        {
+            static void* oldFn_ = nullptr;
+            return &oldFn_;
+        }
+    };
+
```
Comment (Alex0vSky) [inline include/gmock-win32.h:68]: When using inheritance, there are not many macro edits.

**Review by Alex0vSky — APPROVED** [review]

File: include/gmock-win32.h (hunk 1 above)
Comment (smalti) [reply include/gmock-win32.h:68]: I agree, maybe we need to try to reduce the code by using a base structure (but it is not easy when there are many arguments)

**Pull request merged**